mod-record-specifications-validator/src/main/java/org/folio/rspec/validation/validator/marc/utils/TagsMatcher.java

```diff
@@ -17,4 +18,8 @@ public static boolean matchesValidTag(String tag) {
   public static boolean matches1xx(String tag) {
     return tag != null && TAG_1XX_PATTERN.matcher(tag).matches();
   }
+
+  public static boolean matchesValidIndicator(String symbol) {
+    return symbol != null && INDICATOR_PATTERN.matcher(symbol).matches();
```

Comment:
I think will be better to NOT use pattern matching as check could be done much easier by verifying that character is in correct ranges.